composer.json

```
@@ -27,9 +27,17 @@
             "Composer\\Config::disableProcessTimeout",
             "docker compose up"
         ],
+        "startPods": [
```

```suggestion
        "k8s-start": [
```
Might be a clearer name because the deployment creates more than just pods in Kubernetes.